extensions/kafka/src/main/java/io/deephaven/kafka/publish/PublishToKafka.java

```diff
@@ -124,28 +80,31 @@ public PublishToKafka(
      * @param topic The destination topic
      * @param keyColumns Optional array of string column names from table for the columns corresponding to Kafka's Key
      *        field.
-     * @param keySpecSerializer Optional {@link Serializer} to use for keys
-     * @param keySerializer Optional {@link KeyOrValueSerializer} to produce Kafka record keys
+     * @param keySerializer Optional {@link Serializer} to use for keys
+     * @param keyChunkSerializer Optional {@link KeyOrValueSerializer} to produce Kafka record keys
```

Comment:
Maybe `kafkaKeySerializer`? `producerKeySerializer`? We need an explanation in the docs of how these things interact, IMO.